core/src/main/java/hudson/cli/BuildCommand.java

```diff
@@ -158,7 +158,7 @@ protected int run() throws Exception {
 
         if (!job.isBuildable()) {
             String msg = Messages.BuildCommand_CLICause_CannotBuildUnknownReasons(job.getFullDisplayName());
-            if (job instanceof AbstractProject<?, ?> && ((AbstractProject<?, ?>)job).isDisabled()) {
+            if (job instanceof ParameterizedJobMixIn.ParameterizedJob && ((ParameterizedJobMixIn.ParameterizedJob) job).isDisabled()) {
```

Comment:
Not 100% clear. Would be preferable to also have it on the `Job` level directly or in `BuildableItem`. But better than it was before